README.md

```diff
@@ -1,103 +1,130 @@
+# Function Extractor
+
 [![Build Status](https://travis-ci.com/philips-software/functiondefextractor.svg?branch=master)](https://travis-ci.com/philips-software/functiondefextractor)
```

Comment:
Tag is of travis, please  consider adding github action tag

Reply:
Corrected